projects/plugins/jetpack/_inc/lib/class.jetpack-iframe-embed.php

```diff
@@ -24,33 +33,51 @@ static function init() {
 		add_filter( 'shortcode_atts_audio', array( 'Jetpack_Iframe_Embed', 'disable_autoplay' ) );
 
 		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			wp_enqueue_script( 'jetpack-iframe-embed', WPMU_PLUGIN_URL . '/jetpack-iframe-embed/jetpack-iframe-embed.js', array( 'jquery' ) );
+			wp_enqueue_script(
+				'jetpack-iframe-embed',
+				WPMU_PLUGIN_URL . '/jetpack-iframe-embed/jetpack-iframe-embed.js',
+				array( 'jquery' ),
+				JETPACK__VERSION,
+				false
+			);
 		} else {
 			$ver = sprintf( '%s-%s', gmdate( 'oW' ), defined( 'JETPACK__VERSION' ) ? JETPACK__VERSION : '' );
-			wp_enqueue_script( 'jetpack-iframe-embed', '//s0.wp.com/wp-content/mu-plugins/jetpack-iframe-embed/jetpack-iframe-embed.js', array( 'jquery' ), $ver );
+			wp_enqueue_script(
+				'jetpack-iframe-embed',
+				'//s0.wp.com/wp-content/mu-plugins/jetpack-iframe-embed/jetpack-iframe-embed.js',
+				array( 'jquery' ),
+				$ver,
+				false
+			);
 		}
 		wp_localize_script( 'jetpack-iframe-embed', '_previewSite', array( 'siteURL' => get_site_url() ) );
 	}
 
-	static function is_embedding_in_iframe() {
+	/**
+	 * Check that we are in an iFrame.
+	 *
+	 * @return bool
+	 */
+	private static function is_embedding_in_iframe() {
 		return (
-			self::has_iframe_get_param() && (
-				self::has_preview_get_param() ||
-				self::has_preview_theme_preview_param()
+			// phpcs:disable WordPress.Security.NonceVerification.Recommended
+			self::has_get_param( $_GET['iframe'] )
+			&& (
+				self::has_get_param( $_GET['preview'] )
+				|| self::has_get_param( $_GET['theme_preview'] )
 			)
+			// phpcs:enable WordPress.Security.NonceVerification.Recommended
 		);
 	}
 
-	private static function has_iframe_get_param() {
-		return isset( $_GET['iframe'] ) && $_GET['iframe'] === 'true';
-	}
-
-	private static function has_preview_get_param() {
-		return isset( $_GET['preview'] ) && $_GET['preview'] === 'true';
-	}
-
-	private static function has_preview_theme_preview_param() {
-		return isset( $_GET['theme_preview'] ) && $_GET['theme_preview'] === 'true';
+	/**
+	 * Check if URL has specific parameter.
+	 *
+	 * @param string $parameter Parameter to check for.
+	 * @return bool
+	 */
+	private static function has_get_param( $parameter ) {
+		return isset( $parameter ) && 'true' === $parameter;
```

Comment:
The `isset()` here does nothing, and in particular won't suppress the "undefined array key" warning in the calls.

You could get around that by making it `&$parameter` on line 79 (and still drop the `isset()` here, it'd be equivalent to `null !== $parameter` which is redundant to the `'true' === $parameter` following), at the cost of it implicitly doing `$_GET[ $parameter ] = null` when it wasn't set previously.

Or you could just inline this above (including the isset), it's not that much code to repeat there.

Reply:
> you could just inline this above (including the isset), it's not that much code to repeat there.

Yeah, that's a good point, it really isn't too bad. 9085789a70791ef406367897389549d7a52b6da7